### PR TITLE
Move to "UseDeconstruction" to `VerifyCS`

### PR DESCRIPTION
**Pull request by DoctorKrolic**

### CODE REVIEW

File: src/Analyzers/CSharp/Tests/UseDeconstruction/UseDeconstructionTests.cs [hunk 1]
```diff
@@ -3,818 +3,914 @@
...
-    {
-        (int name, int age) [|t1|] = GetPerson();
-        Console.WriteLine(t1.name + "" "" + t1.age);
-    }
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    void M()
+                    {
+                        (string name, int age) [|t1|] = GetPerson();
```
Comment (DoctorKrolic) [inline src/Analyzers/CSharp/Tests/UseDeconstruction/UseDeconstructionTests.cs:126]: There is a `TestTupleTypeInForEach`, that verifies the same thing for `foreach` and it doesn't have tuple type mismatch. So I consider type mismatch here as unintended noise error

File: src/Analyzers/CSharp/Tests/UseDeconstruction/UseDeconstructionTests.cs [hunk 2]
```diff
@@ -3,818 +3,914 @@
...
-    {
-        /*1*/(/*2*/int/*3*/ name, /*4*/int/*5*/ age)/*6*/ [|t1|] = GetPerson();
-        Console.WriteLine(/*7*/t1.name/*8*/ + "" "" + /*9*/t1.age/*10*/);
-    }
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    void M()
+                    {
+                        /*1*/(/*2*/string/*3*/ name, /*4*/int/*5*/ age)/*6*/ [|t1|] = GetPerson();
```
Comment (DoctorKrolic) [inline src/Analyzers/CSharp/Tests/UseDeconstruction/UseDeconstructionTests.cs:504]: This test case is for trivia, so I consider tuple type mismatch here an unintended noise error

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by DoctorKrolic** [comment]
@CyrusNajmabadi Restart failed CI pls

**Comment by DoctorKrolic** [comment]
@CyrusNajmabadi Restart failed CI pls

**Pull request merged**